### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by dariia-romanova**
[DEMO LINK](https://dariia-romanova.github.io/js_task_generate_table_DOM/)

### CODE REVIEW

**Review by SerhiiKirik — APPROVED** [review]
Interesting approach

Good job 🚀

File: src/scripts/main.js [hunk 1]
```diff
@@ -1,8 +1,32 @@
 'use strict';
 
 const people = require('./lib/people');
+const table = document.querySelector('.dashboard');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+for (const person of people) {
```
Comment (AlinaMatuschak) [inline src/scripts/main.js:6]:
```suggestion
for (const { name, sex, born, died } of people) {
```

File: src/scripts/main.js [hunk 2]
```diff
@@ -1,8 +1,32 @@
...
+  const age = document.createElement('td');
+  const century = document.createElement('td');
+
+  fullName.textContent = person.name;
+
+  if (person.sex === 'f') {
+    gender.textContent = 'Female';
+  } else {
+    gender.textContent = 'Male';
+  }
```
Comment (AlinaMatuschak) [inline src/scripts/main.js:22]: you can use ternary operator

**Review by AlinaMatuschak — APPROVED** [review]
Good work 💪

**Pull request closed without merging**